modules/ingestor/src/graph/sbom/common/license.rs

```diff
@@ -14,21 +14,12 @@ const NAMESPACE: Uuid = Uuid::from_bytes([
 #[derive(Default, Debug, Clone, PartialEq, Eq)]
 pub struct LicenseInfo {
     pub license: String,
-    pub refs: HashMap<String, String>,
 }
 
 impl LicenseInfo {
     pub fn uuid(&self) -> Uuid {
-        let mut text = self.license.clone();
```

Comment:
In this hashmap, the key corresponds to the licenseId in hasExtractedLicensingInfos, and the value corresponds to the name. The licenseId is only valid within the current SBOM file. The name is described in the specification as "Provide a common name of the license that is not on the SPDX list. If the License Name field is not present for a license, it implies an equivalent meaning to NOASSERTION.". Using the name as a UUID better reflects its uniqueness.
https://spdx.github.io/spdx-spec/v2.3/other-licensing-information-detected/#1031-description